CMakeLists.txt

```diff
@@ -414,6 +431,10 @@ if (UNWIND_LIBRARY)
   target_link_libraries (glog PUBLIC ${UNWIND_LIBRARY})
 endif (UNWIND_LIBRARY)
 
+if (HAVE_DBGHELP)
+   target_link_libraries (glog PUBLIC DbgHelp)
```

Comment:
At least with MSVC, this isn't necessary due to
```
#pragma comment(lib, "DbgHelp")
```
from [here](https://github.com/google/glog/blob/5cfa9a28f51c41bcafd993f6e93a28daf977d276/src/demangle.cc#L43). But I'm guessing it's needed for MinGW / Cygwin? Let me get one installed and give it a whirl.